### DISCUSSION TITLE
Extending storage

### DISCUSSION THREAD

**Discussion by poxenham-myp** [datalust/seq-tickets#2239] [Help]
Hi there,

We would like to add more storage to our Seq server which is running on a VM in Azure. Is it possible to add another disk and have the storage use the current location as well as the new disk?

If not is there a best practice or and considerations for how to extend the storage?

Thanks

**Comment by liammclennan** [comment]
Hi @poxenham-myp 

Seq does not have support for this. It is best to deal with it at the disk level with JBOD or RAID.

**Comment by poxenham-myp** [comment]
That's what I figured but thought I'd just double check. Thanks


PETER OXENHAM

Chief Technology Officer | MYP Corporation Pty Ltd
BInfoTech

***@***.******@***.***> | +61 7 3393 2833
Corporate & Client Services: 14/8 Metroplex Avenue Murarrie Qld 4172
Technical & Development:     11/8 Metroplex Avenue Murarrie Qld 4172

[https://vbox.mypcorp.com/signatures/myp-Logo-Horizontal.png]

[Facebook]<https://www.facebook.com/MYPCorp>

[Linkedin]<http://www.linkedin.com/company/myp-corporation>

[Twitter]<https://twitter.com/mypcorp>

[YouTube]<https://www.youtube.com/user/MYPCorporation>

[myCommunity]<https://www.facebook.com/groups/2072086089501953>

mypcorp.com.au<https://mypcorp.com.au/>

In accordance with the Spam Act 2003, we offer you the opportunity to UNSUBSCRIBE to any further communication from MYP Corporation Pty Ltd ABN 37 131 264 465. To unsubscribe, please reply with 'unsubscribe' in the subject line. The information transmitted is intended only for the person or entity to which it is addressed and may contain confidential and/or privileged material. Any review, re-transmission, dissemination or other use of, or taking of any action in reliance upon, this information by persons or entities other than the intended recipient is prohibited. If you receive this in error, please contact the sender and destroy or delete the material from any computer. MYP Corp does not authorise or condone the sending of any abusive, obscene, defamatory, illegal, or offensive material contained in this email message and therefore accepts no liability for its inclusion. Further, MYP Corp cannot guarantee that this email message is secure and will not be intercepted by a third party.

________________________________

**Reply by liammclennan** [reply]
You're welcome. 

If you end up migrating to a larger disk just make sure Seq is stopped before you try to move its files.
